beacon-chain/core/state/state_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestGenesisBeaconState_OK(t *testing.T) {
-	if params.BeaconConfig().SlotsPerEpoch != 64 {
+	if params.BeaconConfig().SlotsPerEpoch != 32 {
 		t.Errorf("SlotsPerEpoch should be 64 for these tests to pass")
```

Comment:
```suggestion
		t.Errorf("SlotsPerEpoch should be 32 for these tests to pass")
```
I think you can just delete this assertion. Seems like it had something to do with shard count